### PR TITLE
draft3 backwards compat fixes, part 1

### PR DESCRIPTION
**Pull request by ChALkeR**

### CODE REVIEW

**Comment by ChALkeR** [comment]
This is not complete in a sense that there are other incompatibilities apart from those two, but we can land this first and decide if we want the other ones later.

File: index.js [hunk 1]
```diff
@@ -579,19 +579,20 @@ const compile = function(schema, cache, root, reporter, opts) {
       consume('oneOf')
     }
 
-    if (node.multipleOf !== undefined) {
-      if (!Number.isFinite(node.multipleOf)) throw new Error('Invalid multipleOf')
+    const multipleOf = node.multipleOf !== undefined ? 'multipleOf' : 'divisibleBy' // draft3 support
```
Comment (kklash) [inline index.js]: Nit: double negatives are confusing
```suggestion
    const multipleOf = node.multipleOf === undefined ? 'divisibleBy' : 'multipleOf' // draft3 support
```

Comment (ChALkeR) [reply index.js]: Fixed, thanks!

Comment (ChALkeR) [reply index.js]: Oh, this was a commitable suggestion -- missed that, sorry ).

**Review by kklash — APPROVED** [review]
utACK

**Pull request merged**